feature-libs/checkout/base/root/model/checkout-step.model.ts

```diff
@@ -5,6 +5,16 @@ export const enum CheckoutStepType {
   REVIEW_ORDER = 'reviewOrder',
 }
 
+/**
+ * TODO:#checkout - either:
```

Comment:
Yep.
But we might not need an enum for it. It could be just a method in the repl facade `isReplOrder`
Or we augment it. Let's discuss in your ticket.

Reply:
as mentioned earlier today, we're keeping the enum :d